InflateException when running the version of Plaid currently on the Play Store

Seems like it is 0.9.11? Here's the stack trace.

This is on a Pixel 2 XL running API 27.

```
W WindowManager: Token{2b8de82 ActivityRecord{b9ac5cd u0 io.plaidapp/.Launcher t13178}} failed creating starting window
W WindowManager: android.view.InflateException: Could not instantiate class android.transition.Transition class io.plaidapp.ui.transitions.Pop
W WindowManager: 	at android.transition.TransitionInflater.createCustom(TransitionInflater.java:224)
W WindowManager: 	at android.transition.TransitionInflater.createTransitionFromXml(TransitionInflater.java:171)
W WindowManager: 	at android.transition.TransitionInflater.createTransitionFromXml(TransitionInflater.java:185)
W WindowManager: 	at android.transition.TransitionInflater.inflateTransition(TransitionInflater.java:79)
W WindowManager: 	at com.android.internal.policy.PhoneWindow.getTransition(PhoneWindow.java:2790)
W WindowManager: 	at com.android.internal.policy.PhoneWindow.installDecor(PhoneWindow.java:2743)
W WindowManager: 	at com.android.internal.policy.PhoneWindow.getDecorView(PhoneWindow.java:2067)
W WindowManager: 	at com.android.server.policy.PhoneWindowManager.addSplashScreen(PhoneWindowManager.java:2960)
W WindowManager: 	at com.android.server.wm.SplashScreenStartingData.createStartingSurface(SplashScreenStartingData.java:55)
W WindowManager: 	at com.android.server.wm.AppWindowContainerController.lambda$-com_android_server_wm_AppWindowContainerController_4735(AppWindowContainerController.java:145)
W WindowManager: 	at com.android.server.wm.-$Lambda$aEpJ2RCAIjecjyIIYTv6ricEwh4.$m$4(Unknown Source:4)
W WindowManager: 	at com.android.server.wm.-$Lambda$aEpJ2RCAIjecjyIIYTv6ricEwh4.run(Unknown Source:27)
W WindowManager: 	at android.os.Handler.handleCallback(Handler.java:790)
W WindowManager: 	at android.os.Handler.dispatchMessage(Handler.java:99)
W WindowManager: 	at android.os.Looper.loop(Looper.java:164)
W WindowManager: 	at android.os.HandlerThread.run(HandlerThread.java:65)
W WindowManager: 	at com.android.server.ServiceThread.run(ServiceThread.java:46)
W WindowManager: Caused by: java.lang.ClassNotFoundException: Didn't find class "io.plaidapp.ui.transitions.Pop" on path: DexPathList[[],nativeLibraryDirectories=[/data/app/com.android.chrome-T8LOYNLPdc8-b4sV1dnBjA==/lib/arm64, /data/app/com.android.chrome-T8LOYNLPdc8-b4sV1dnBjA==/base.apk!/lib/arm64-v8a, /system/lib64]]
W WindowManager: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:125)
W WindowManager: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
W WindowManager: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
W WindowManager: 	at android.transition.TransitionInflater.createCustom(TransitionInflater.java:210)
W WindowManager: 	... 16 more
```

--- a/app/src/main/res/transition/home_enter.xml
+++ b/app/src/main/res/transition/home_enter.xml
@@ -15,19 +15,4 @@
   limitations under the License.
   -->
 
-<transitionSet
-    xmlns:android="http://schemas.android.com/apk/res/android">
-
-    <transition
-        class="io.plaidapp.ui.transitions.Pop"
-        android:duration="@android:integer/config_shortAnimTime"
-        android:startDelay="@android:integer/config_shortAnimTime"
-        android:interpolator="@android:interpolator/fast_out_slow_in">
-        <targets>
-            <target android:targetId="@id/fab" />
-        </targets>
-    </transition>
-
-    <fade />
-
-</transitionSet>
+<fade />

